### PR TITLE
Notification Hub - fixes

### PR DESCRIPTION
**Pull request by johnpapa**
added intros for heading sections

needs review before #sign-off

### CODE REVIEW

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180122134921-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-services/notification-hubs.md [hunk 1]
```diff
@@ -30,13 +31,16 @@ Notification Hubs works great for both enterprise and consumer scenarios. Here a
 
 ### Install npm package
 
-Install the Azure Noticiation Hubs module for Node.js
+Install the Azure Noticiation Hubs module
```
Comment (clarkio) [inline docs-ref-services/notification-hubs.md]: Update to read `npm module`

File: docs-ref-services/notification-hubs.md [hunk 2]
```diff
@@ -30,13 +31,16 @@ Notification Hubs works great for both enterprise and consumer scenarios. Here a
 
 ### Install npm package
```
Comment (clarkio) [inline docs-ref-services/notification-hubs.md]: Update to `Install npm module`

**Comment by johnpapa** [comment]
#sign-off

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180352061059-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180353015164-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180402242124-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180416437118-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by herohua** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-services/notification-hubs.md](https://github.com/MicrosoftDocs/azure-docs-sdk-node/blob/johnpapa-patch-13/docs-ref-services/notification-hubs.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C7%5C18%5Cb6261451-6613-d51e-95b5-8b5dc96f3a2c%5CPullRequest%5C201707180448546430-113%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by TomArcher** [comment]
@johnpapa @clarkio @jawache merged

**Pull request merged**